src/google/cloud/ndb/query.py

```diff
@@ -934,7 +970,7 @@ def __init__(
         kind=None,
         ancestor=None,
         filters=None,
-        orders=None,
```

Comment:
We still need `orders` in the signature for backwards compatibility. It should just be documented as a deprecated synonym for `order`.